src/components/Header/RightNav.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Ul = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-flow: row nowrap;
+
+  li {
+    padding: 18px 5px;
+    font-size: 1.7rem;
+  }
+
+  @media (max-width: 768px) {
+    flex-flow: column nowrap;
+    background-color: black;
+    position: fixed;
+    transform: ${({ open }) => open ? 'translateX(0)' : 'translateX(100%)'};
+    top: 0;
+    right: 0;
+    height: 100vh;
+    width: 300px;
+    padding-top: 3.5rem;
+    transition: transform 0.3s ease-in-out;
+
+    li {
+      color: #fff;
+    }
+  }
+`;
+
+const RightNav = ({ open }) => {
+  return (
+    <Ul open={open}>
```

Comment:
@koral14 I agree with what @nikkigraybeal has said here. Once that change is made I think this will work really well. The styling looks awesome! I especially like how responsive it is to screen size. You won't know for sure right now if we are logged in or not as that's being worked on in a separate ticket, but if you add the props in as Nikiki suggests, it should be easy to hook up once the authentication context is ready to go.

Reply:
@nikkigraybeal and @jpremo I hope I understood correctly your suggestions. Please notice my code below.